tests/migration/postcopy.go

```diff
@@ -170,7 +170,7 @@ var _ = SIGMigrationDescribe("VM Post Copy Live Migration", func() {
 				podName := "migration-killer-pod-"
 
 				// kill the handler
-				pod := libpod.RenderPrivilegedPod(podName, []string{"/bin/bash", "-c"}, []string{fmt.Sprintf("while true; do pkill -9 virt-handler && sleep 5; done")})
+				pod := libpod.RenderPrivilegedPod(podName, []string{"/bin/bash", "-c"}, []string{"date; pkill -e -9 virt-handler || echo not found"})
```

Comment:
Why is this needed?

Reply:
Which part? The main thing here is getting rid of the loop. Killing virt-handler once should be enough.
The date, `-e` and echo are to help debug failures. Before my change, the script is completely silent.

Reply:
The loop part, I remember I relied on it at least in one test. Please keep it if it is not a fix in which case please apply it only to specific tests.

Reply:
This is used by only one test, and that test just needs virt-handler to disappear once, and in fact needs it to be there after to mark things as failed.
Maybe this function is copied somewhere else and that's what you remember?

Reply:
Correct, thanks!